smoke/config.go

```diff
@@ -217,15 +226,18 @@ func validateIsolationSegments(config *Config) {
 	if !config.EnableIsolationSegmentTests {
 		return
 	}
-	if config.GetBackend() != "diego" {
+	if config.Backend != "diego" {
```

Comment:
I take that back -- it looks like your change is consistent with the rest of `config.go` so it looks good to me.